src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends AbstractFigure {
+    private final double base1;
+    private final double base2;
+    private final double height;
+
+    public IsoscelesTrapezoid(double base1, double base2, double height, String color) {
```

Comment:
Creating a figure, don't pass expressions in the constructor. It's better to create separate variables before the constructor call and pass them on for better code readability.